docs/getting-started/linux-manual.md

```diff
@@ -160,11 +162,40 @@ $ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_REALM=us0 \
 
 > Use of a SemVer tag over `latest` is highly recommended.
 
-In the case of Docker, a volume mount may be required to load custom
-configuration as shown above.
+In the case of Docker, a volume mount may be required to load the custom
+configuration file as shown above.
 
 If the custom configuration includes a `memory_limiter` processor then the
 `ballast_size_mib` parameter should be the same as the
 `SPLUNK_BALLAST_SIZE_MIB` environment variable. See
 [gateway_config.yaml](../../cmd/otelcol/config/collector/gateway_config.yaml)
 as an example.
+
+Using `SPLUNK_CONFIG_YAML` instead of `SPLUNK_CONFIG` would be something like:
+
+```bash
+$ SPLUNK_CONFIG_YAML="receivers:\n   hostmetrics:\n      collection_interval: 10s\n      scrapers:\n         memory:\nexporters:\n   otlp:\n      endpoint: localhost:23456\n      insecure: true\nservice:\n   pipelines:\n      metrics:\n         receivers: [hostmetrics]\n         exporters: [otlp]"
+
+$ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 \
+    -e SPLUNK_REALM=us0 \
+    -e SPLUNK_CONFIG_YAML=${SPLUNK_CONFIG_YAML} \
+    -p 23456:23456 \
+    --name otelcol quay.io/signalfx/splunk-otel-collector:latest
+```
+Where `SPLUNK_CONFIG_YAML` is the single-line version of the multi-line configuration YAML below:
```

Comment:
I feel like if we aren't demonstrating how to actually set this is an env var we create a strong possibility for repeated questions.

Reply:
I edited this sentence hoping to make it clearer.